scripts/error-handler.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @description Error handling
+ */
+;(function() {
+    const errorHandler = {};
+
+    errorHandler.clear = () => {
+        document.getElementById('notifications').innerHTML = '';
+    };
+
+    errorHandler.escape = (str) => {
+          return str
+            .replace(/&/g, '&')
+            .replace(/'/g, "'")
+            .replace(/"/g, '"')
+            .replace(/>/g, '>')
+            .replace(/</g, '<')
+            .replace(/\//g, '/');
+    };
+
+    /**
+     * Adds an error message to the notifications box
+     * @param  {[type]} message Error message.
+     * @param  {[type]} type    one of 'danger', 'warning', 'success', 'info', 'link', 'primary', or empty
+     */
+    errorHandler.addError = (message, type) => {
+        const errorBox = document.createElement('div');
```

Comment:
Interesting how youre building a div and adding stuff to turn it into a notification